tests/unit/test_proxy.py

```diff
@@ -1,61 +1,162 @@
-from PyRDF.Proxy import Proxy
+from PyRDF.Proxy import ActionProxy, TransformationProxy
 from PyRDF.Node import Node
 from PyRDF.backend.Backend import Backend
 from PyRDF import RDataFrame
 import unittest
 import PyRDF
 
+# As of Python 3, abstract classes that do not implement
```

Comment:
Using `ABCMeta` as a parent class of `Proxy` and the decorator `@abstractmethod` this test should work, can you check with the current implementation?

Reply:
Yes, now the test works properly